components/shared/hexToRgb.js

```diff
@@ -0,0 +1,11 @@
+const hexToRgb = hex => {
```

Comment:
Testing the implementation, i noticed that method is not used in any place.
What do you think to just remove it?

Reply:
Good idea!